docs/server/commands.rst

```diff
@@ -419,6 +419,49 @@ Available options:
   (quotes included).
 
 
+.. django-admin:: set_filetype
+
+set_filetype
+^^^^^^^^^^^^
+
+.. versionadded:: 2.8
+
+This command sets file formats for projects, and also allows to convert files
+to another format.
+
+.. django-admin-option:: --from-filetype
+
+Specifies to convert stores of this file type.
+
+.. django-admin-option:: --matching
+
+Specifies which stores to convert by matching their path within the project,
+excluding file extension.
+
+`--from-filetype` and `--matching` can be both specified at the same time.
+
+
+For example, to add the `properties` format to a project, run:
+
+.. code-block:: console
+
+    $ pootle set_filetype --project=myproj properties
+
+
+To convert stores of `po` format to `properties`, run:
+
+.. code-block:: console
+
+    $ pootle set_filetype --project=myproj --from-filetype=po properties
+
+
+For example, to add a new format to a project, run:
+
+.. code-block:: console
+
+    $ pootle set_filetype --project=myproj --matching=mydir/myfile-* properties
```

Comment:
First example and this one achieve different things. The only way they would behave the same is if in the project all files that don't have the specified target filetype (`properties`) are exactly the files in the project that match the given `--matching` glob.